__tests__/app.test.js

```diff
@@ -0,0 +1,72 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index");
+const devData = require("../db/data/development-data/index.js");
```

Comment:
i was trying to use it earlier just to make sure this is how we change which dataset were accessing but i have removed it now since we only use the test data in the tests